ui/app/styles/core/buttons.scss

```diff
@@ -34,10 +37,54 @@ $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
     color: darken($grey-dark, 10%);
   }
 
-  &.is-inverted.is-outlined {
+  &:disabled {
+    background-color: $white;
+    border-color: #dbdbdb;
+    box-shadow: none;
+    opacity: 0.5;
+  }
+
```

Comment:
For context: this is the button in question. ![image](https://user-images.githubusercontent.com/6618863/227680412-8e61d8b9-dc8e-41af-b7cb-9863ab7717df.png)